src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,37 @@
+import "./Introduce.scss";
+import { Link, Navigate, UseNavigate, useNavigate } from "react-router-dom";
+
+const Introduce = () => {
+  const navigate = useNavigate();
+  const goToProduct = () => {
+    navigate("/product");
+  };
+  const handleInput = () => {
+    console.log("input 태그 동작");
+  };
+  const handleButton = () => {
+    console.log("button 태그 동작");
+  };
+  return (
+    <main>
+      <div className="introduce">
+        <div className="card">wecode</div>
+        <p className="name">장건웅</p>
+        <p className="yearTh">46기</p>
+        <div className="form-wrap">
+          <input className="introduce-input" onChange={handleInput} />
+          <button
+            className="introduce-button"
+            value="버튼"
+            onClick={(handleButton, goToProduct)}
```

Comment:
```suggestion
            onClick={handleButton}
```

동시에 두 함수를 발생하게 하는 것은 좋지 못한 방법입니다!
하나의 함수에서 동작 될 수 있도록 구현해주세요